app/views/registeredAddressForEori.scala.html

```diff
@@ -31,7 +32,9 @@
 
         @components.error_summary(form.errors)
 
-        @components.heading("registeredAddressForEori.heading")
+        @components.heading(
```

Comment:
for some reason the EORI number is not shown here... Any tip to fix this?

Reply:
I think the problem is that `heading.scala` expects a messages key, not a messages value.
Can we make `heading.scala` more generic so that it takes also a sequence of optional String arguments to pass when calling `messages`?

Reply:
I tried making `heading.scala` more generic, but I can't make the test passing...